draft-ietf-ccwg-bbr.md

```diff
@@ -333,6 +333,11 @@ transport protocol payload and headers, implementations MAY include
 in C.SMSS the size of other headers, such as network-layer or
 link-layer headers.
 
+C.has_selective_acks: True if the connection has the capability to receive
+selective acknowledgments, e.g., via a QUIC ACK Range {{RFC9000}} or
```

Comment:
selective acknowledgements seems a bit odd for QUIC, but I am struggling to find a better term.  discontiguous acknowledgements is correct, but maybe not any better?

Reply:
selective_ack is a common term - so I quite like that personally: Could we just explicitly note that QUIC always supports this using the QUIC ACK Range?